src/libraries/System.Drawing.Common/src/System/Drawing/Graphics.Windows.cs

```diff
@@ -682,40 +684,59 @@ public void DrawIconUnstretched(Icon icon, Rectangle targetRect)
         /// WARNING: This method is for internal FX support only.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
+        [Obsolete(DiagnosticId = "SYSLIB0016", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
```

Comment:
Ah, sorry -- I should have also mentioned that we're defining these obsoletions here: https://github.com/dotnet/runtime/blob/main/src/libraries/Common/src/System/Obsoletions.cs

The attribute should also include the message (as well as the diagnosticid and urlformat).  Here's an example:

* src: https://github.com/dotnet/runtime/blob/01b7e73cd378145264a7cb7a09365b41ed42b240/src/libraries/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/HashAlgorithm.cs#L21
* ref: https://github.com/dotnet/runtime/blob/01b7e73cd378145264a7cb7a09365b41ed42b240/src/libraries/System.Security.Cryptography.Primitives/ref/System.Security.Cryptography.Primitives.cs#L20